packages/app-js/src/LocalStorage.tsx

```diff
@@ -0,0 +1,106 @@
+// Copyright 2017-2019 @polkadot/app-js authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { BareProps, I18nProps } from '@polkadot/ui-app/types';
+
+import React from 'react';
+import { Button as SUIB, Popup } from 'semantic-ui-react';
+import { Button, Input } from '@polkadot/ui-app/index';
```

Comment:
The `ui-app` `<Button />` doesn't work well together with the `<Popup />` Component, the event triggers are not working properly.  I was trying to debug it, but then went for the easy fix of just using the original Semantic UI components...

Reply:
That is weird, since it is basically just a wrapper. So no ideas. 

Just put a FIXME and let somebody else take a look. No need to continue bumping heads on it, the workaround works.